perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -4519,6 +4515,16 @@ public void convertAttributeValuesToUnique(PerunSession session, AttributeDefini
 		}
 	}
 
+	@Override
+	public int convertAttributeValuesToNonunique(PerunSession session, AttributeDefinition attrDef) {
+		String tablePrefix = attributeToTablePrefix(attrDef);
+		try {
+			return jdbc.update("DELETE FROM " + tablePrefix + "_attr_u_values WHERE attr_id=?", attrDef.getId());
```

Comment:
@stavamichal Do we really want to delete all values when doing this conversion? Couldn't we just move those values to a different table? I understand that when you do the opposite conversion, you have to remove the old values because there could be duplicates. But in this direction, I think it is not necessary.

Reply:
Unique attributes has values in both tables "_attr_values" and "_attr_u_vaues". So yes, you need to delete them. If you let them here, you can have a big problem when you want to convert attribute to unique again. There will be inconsistency in data.

Reply:
I see. I thought they are only in this table. Thanks for your explanation.